pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -533,7 +463,7 @@ func (c *Controller) Services() ([]*model.Service, error) {
 		out = append(out, svc)
 	}
 	c.RUnlock()
-	sort.Slice(out, func(i, j int) bool { return out[i].Hostname < out[j].Hostname })
+	sort.Slice(out, func(i, j int) bool { return out[i].CreationTime.Before(out[j].CreationTime) })
```

Comment:
I think this may change behavior in some cases, any motivation for this?

Reply:
There are many sorting using create time, and i think this is a bug here